python-project-template/src/{{module_name}}/__init__.py.jinja

```diff
@@ -1,3 +1,3 @@
 {%- if create_example_module -%}
 from .example_module import *
-{% endif %}
\ No newline at end of file
+{% endif -%}
```

Comment:
Just to confirm, you checked this either manually or with CI for the following four cases of hydrated templates:

- pylint with example module
- pylint without example module
- black with example module
- black without example module

Reply:
All of these passed the CI checks when I committed. Is that sufficient?
I do not trust my local testing as it is easy to make mis steps.

Reply:
A new PR has been merged that expands the CI to include more cases (we avoided doing this in the past because of how quickly this can explode the number of tests) but for here and now, we might as well make use of it. 

I'm not sure how to get this PR to run all 12 test cases. Hopefully there's a nicer way that the brute force approach of closing and recreating the PR.

Reply:
I made a gratuitous whitespace change and forced the rebuild. I think we might add this line to ci.yml:

on: workflow_dispatch

Reportedly enables manual running actions
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch